Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM tomcat:7-jre8
+FROM tomcat:jre8
```

Comment:
This should be explicit about the major Tomcat version.  I.e. tomcat:8-jre8.  Otherwise we may jump up to the next major version of Tomcat without any notice (or testing).